tests/test_transforms/test_feature_selection/test_filter_transform.py

```diff
@@ -97,3 +97,105 @@ def test_exclude_filter_wrong_column(ts_with_features):
     transform = FilterFeaturesTransform(exclude=["non-existent-column"])
     with pytest.raises(ValueError, match="Features {.*} are not present in the dataset"):
         ts_with_features.fit_transform([transform])
+
+
+@pytest.mark.parametrize("return_features", [True, False])
+@pytest.mark.parametrize(
+    "columns, saved_columns",
+    [
+        ([], ["target", "exog_1", "exog_2"]),
+        (["target"], ["exog_1", "exog_2"]),
+        (["exog_1", "exog_2"], ["target"]),
+        (["target", "exog_1", "exog_2"], []),
+    ],
+)
+def test_transform_exclude_save_columns(ts_with_features, columns, saved_columns, return_features):
+    original_df = ts_with_features.to_pandas()
+    transform = FilterFeaturesTransform(exclude=columns, return_features=return_features)
+    ts_with_features.fit_transform([transform])
+    df_transformed = ts_with_features.df
+    got_columns = set(df_transformed.columns.get_level_values("feature"))
+    assert got_columns == set(saved_columns)
+
+    segments = ts_with_features.segments
+    for column in got_columns:
+        assert np.all(
+            df_transformed.loc[:, pd.IndexSlice[segments, column]]
+            == original_df.loc[:, pd.IndexSlice[segments, column]]
+        )
+
+
+@pytest.mark.parametrize("return_features", [True, False])
+@pytest.mark.parametrize(
+    "columns, saved_columns",
+    [
+        ([], []),
+        (["target"], ["target"]),
+        (["exog_1", "exog_2"], ["exog_1", "exog_2"]),
+        (["target", "exog_1", "exog_2"], ["target", "exog_1", "exog_2"]),
+    ],
+)
+def test_transform_include_save_columns(ts_with_features, columns, saved_columns, return_features):
+    original_df = ts_with_features.to_pandas()
+    transform = FilterFeaturesTransform(include=columns, return_features=return_features)
+    ts_with_features.fit_transform([transform])
+    df_transformed = ts_with_features.df
+    got_columns = set(df_transformed.columns.get_level_values("feature"))
+    assert got_columns == set(saved_columns)
+    segments = ts_with_features.segments
+    for column in got_columns:
+        assert np.all(
+            df_transformed.loc[:, pd.IndexSlice[segments, column]]
+            == original_df.loc[:, pd.IndexSlice[segments, column]]
+        )
+
+
+@pytest.mark.parametrize(
```

Comment:
For these tests you also need to check that in case of `return_features=False` inverse_transform does not affect the dataframe